### PR TITLE
[viessmann] code cleanup

### PR DESCRIPTION
**Pull request by rogrun**
Signed-off-by: Ronny Grun <ronny.grun@t-online.de>

### CODE REVIEW

File: bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/api/ViessmannAuth.java [hunk 1]
```diff
@@ -287,21 +286,20 @@ private void updateBridgeStatusApiKey() {
...
+            logger.debug("InterruptedException on call to Viessmann authorization API: {}", e.getMessage());
         }
         return null;
     }
 
-    private @Nullable String executeUrlToken(String method, String url) {
+    private @Nullable String executeUrlToken(String url) {
         Request request = httpClient.newRequest(url);
         request.timeout(API_TIMEOUT_MS, TimeUnit.MILLISECONDS);
         request.method("POST");
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/api/ViessmannAuth.java]: You can use the predefined const `HttpMethod.POST` here.

File: bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/api/ViessmannAuth.java [hunk 2]
```diff
@@ -256,18 +255,18 @@ private void getRefreshTokens() throws ViessmannAuthException {
...
-                String.format("Login fails. Please check API Key."));
+                "Login fails. Please check API Key.");
     }
 
-    private @Nullable String executeUrlAuthorize(String method, String url) {
+    private @Nullable String executeUrlAuthorize(String url) {
         Request request = httpClient.newRequest(url);
         request.timeout(API_TIMEOUT_MS, TimeUnit.MILLISECONDS);
-        request.method(method);
+        request.method("GET");
```
Comment (cweitkamp) [inline bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/api/ViessmannAuth.java]: You can use predefined const `HttpMethod.GET` here. Furthermore the `Request` can be used like a builder pattern.

For example:
```suggestion
        Request request = httpClient.newRequest(...).timeout.(...).method(...).header(..., ...);
```

**Review by cweitkamp — COMMENTED** [review]
Thanks for the cleanup.

**Review by cweitkamp — APPROVED** [review]
LGTM.

**Review by J-N-K — APPROVED** [review]

**Pull request merged**